integration/mounts/mounts.go

```diff
@@ -0,0 +1,193 @@
+package mounts
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	socketPath     = "/var/run/vc/vm/foobar/console.sock"
+	kataConfigPath = "/usr/share/kata-containers/configuration.toml"
+)
+
+var (
+	kataConfigBackup = strings.Join([]string{kataConfigPath, "bak"}, ".")
```

Comment:
Rather than doing the `cp` dance, you could just `mkdir /etc/kata-containers/`, copy the file in and then `rm -rf /etc/kata-containers/` at the end to avoid modifying the pristine config files.